internal/plans/plan.go

```diff
@@ -197,6 +199,36 @@ func (p *Plan) ProviderAddrs() []addrs.AbsProviderConfig {
 	return ret
 }
 
+// Variable mapper checks that all of the provided variables match what has been provided in the plan
+// They may be sourced from the environment, from cli args, and autoloaded tfvars files
+func (plan *Plan) VariableMapper() configs.StaticModuleVariables {
```

Comment:
This is not strictly needed for this PR, but it was duplicated code that was in the paths this PR modified.  It could be moved to it's own PR if desired.